rules/prefer-key-over-key-code.js

```diff
@@ -0,0 +1,198 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const keys = ['keyCode', 'charCode', 'which'];
+const extraCitation = 'See https://goo.gl/cRK532 for more info.';
```

Comment:
This doesn't belong in the message. People can read about it in the rule docs instead.